### PR TITLE
feat(frontend): add jspdf-react library

### LINKED ISSUE #655: A lifebank user can download a report as PDF of transactions history
Dependent: #555
Must first have: #591

- [x] Content: @ldrojas 
- [ ] Apply to PDF

Considering:
1. https://github.com/apipemc/jspdf-react/blob/master/src/PDF.js
2. https://www.npmjs.com/package/jspdf

### LINKED ISSUE #656: A sponsor user can download a report as PDF of transactions history
Dependent: #555 and #658
See also.: #591 and #655

### PR DESCRIPTION
**Pull request by leisterfrancisco**
Resolves #655
Resolves #656 

### What does this PR do?
Enable the user to create a pdf

### How should this be tested?
- Make run
- Pending

### CODE REVIEW

File: hapi/src/api/account.api.js [hunk 1]
```diff
@@ -17,7 +17,6 @@ const locationApi = require('./location.api')
 const preRegLifebank = require('./pre-register.api')
 const verificationCodeApi = require('./verification-code.api')
 const mailApi = require('../utils/mail')
-const verifyEmailHandler = require('../routes/verify-email/verify-email.handler')
```
Comment (leisterfrancisco) [inline hapi/src/api/account.api.js:20]: unused variable

**Review by Edgar097 — APPROVED** [review]
Looks really nice!

File: webapp/src/language/en.json [hunk 2]
```diff
@@ -497,6 +497,22 @@
       "Oct": "October",
       "Nov": "November",
       "Dec": "December"
+    },
+    "report": {
+      "from": "from",
+      "to": "to",
+      "pdfHeader": "History of transactions",
+      "pdfFooter": "Lifebank is an open-source platform that uses blockchain technology to incentivizeblood donations. For more information about Lifebank, go to http://lifebank.io/.",
```
Comment (JustinCast) [inline webapp/src/language/en.json:505]: fix "incentivizeblood"

File: webapp/src/language/es.json [hunk 3]
```diff
@@ -496,6 +496,22 @@
       "Oct": "Octubre",
       "Nov": "Noviembre",
       "Dec": "Diciembre"
+    },
+    "report": {
+      "from": "de",
+      "to": "a",
+      "pdfHeader": "Historial de transacciones",
+      "pdfFooter": "Lifebank es una plataforma de código abierto que usa tecnología blockchain paraincentivar las donaciones de sangre. Para más información sobre Lifebank, visitehttp://lifebank.io/.",
```
Comment (JustinCast) [inline webapp/src/language/es.json:504]: Fix "paraincentivar"

**Review by JustinCast — CHANGES_REQUESTED** [review]

File: webapp/src/components/TransactionReport/TransactionReport.js [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+      doc.text(t('report.pdfFooter'), pageWidth/2, pageHeight - 20, { align: 'center', maxWidth: pageWidth - 50 })
+    }
+
+    doc.save(t('report.reportDownloadName'))
+    onReportSaved()
+  }
+
+  return (
+    <>
+    </>
```
Comment (JustinCast) [inline webapp/src/components/TransactionReport/TransactionReport.js:165]: Is this ok?

**Review by JustinCast — APPROVED** [review]

**Review by steph222 — APPROVED** [review]

**Pull request merged**